seimas/accounts/views.py

```diff
@@ -8,16 +8,20 @@
 import seimas.accounts.helpers.allauth as allauth_helpers
 import seimas.accounts.forms as accounts_forms
 from seimas.website.helpers import formrenderer
-
+#from allauth.account.auth_backends.AuthenticationBackend import AuthenticationBackend
 
 def login(request):
+    form1 = accounts_forms.LoginForm(request.POST)
```

Comment:
And read [documentation](https://docs.djangoproject.com/en/1.9/topics/forms/#the-view) to understand, how to process forms properly. In your case, you assume, that incoming request is a `POST` request. You should check request type, before constructing [bound form](https://docs.djangoproject.com/en/1.9/ref/forms/api/#ref-forms-api-bound-unbound).